cpp/models/abm/world.h

```diff
@@ -51,23 +54,92 @@ class World
 
     /**
      * @brief Create a World.
-     * @param[in] params Parameters of the Infection that are the same everywhere in the World.
+     * @param[in] num_agegroups The number of age groups in the simulated world.
      */
-    World(const GlobalInfectionParameters& params = {})
-        : m_infection_parameters(params)
-        , m_migration_parameters()
+    World(size_t num_agegroups)
+        : parameters(num_agegroups)
         , m_trip_list()
         , m_cemetery_id(add_location(LocationType::Cemetery))
     {
         use_migration_rules(true);
     }
 
+    /**
+     * @brief Create a copied World.
+     * @param[in] other The World that needs to be copied. 
+     */
+    World(const World& other)
+        : parameters(other.parameters)
+        , m_persons()
+        , m_locations()
+        , m_trip_list(other.m_trip_list)
```

Comment:
Is it ensured that trip_list gets **copied** correctly? (I think we dont have any test for that)

Reply:
I now added the tests for trips.